jiant/tasks/tasks.py

```diff
@@ -1416,19 +1416,19 @@ def load_data(self):
         targ_map = {"not_entailment": 0, "entailment": 1}
 
         self.train_data_text = load_pair_nli_jsonl(
-            os.path.join(self.path, "winogender_filtered.jsonl"),
+            os.path.join(self.path, "Winogender.jsonl"),
```

Comment:
I think it should be WInoGender no? (at least how Amanpreet had named the file).

Reply:
I told him to use `Winogender` (that's how the paper refers to itself)

Reply:
Alright - currently it isn't named that way